AirCasting/Services/DownloadMeasurmentsService.swift

```diff
@@ -83,4 +88,18 @@ final class DownloadMeasurementsService: MeasurementUpdatingService {
             }
         })
     }
+    
+    private func calculateLastSync(sessionEndTime: Date?, lastMeasurementTime: Date?) -> Date {
```

Comment:
this function is a great candidate to be a standalone module (object). It could then be unit tested properly. I also feel that all this could be done simpler using SDKs `Date` methods instead of converting to seconds.
It would be a great exercise to refactor this in the future using TDD approach, maybe after we ship we can schedule a session where I'd show you how to do this

Reply:
I moved this func to separate class - that's what you meant by _standalone module (object)_, right? :) 
I'd be happy to pair-programme this refactor with you, I'll put a reminder somewhere in postMVP todo column :)